src/components/layout/nav.js

```diff
@@ -12,7 +12,7 @@ import { SharedLinkStyle } from 'components/localization/localized-link'
 import Login from 'common/login'
 import device from 'themes/device'
 // Icons
-import LogoBeta from 'images/svg/logo-beta.svg'
+import LogoBeta from 'images/svg/logo_deriv.svg'
```

Comment:
can you change the LogoBeta as well, and for file name you can youse `kebab-case`